sympy/ntheory/ec.py

```diff
@@ -0,0 +1,374 @@
+from sympy import symbols
```

Comment:
This could be `from sympy.core.symbol import symbols` and placed after `core.relational`.